main.go

```diff
@@ -23,29 +23,34 @@ func main() {
 				Name: "start",
 				Flags: []cli.Flag{
 					&cli.IntFlag{
-						Name:  "port",
-						Usage: "port to serve requests on",
-						Value: 8080,
+						Name:    "port",
+						Value:   8080,
+						EnvVars: []string{"SOMEGUY_PORT"},
+						Usage:   "port to serve requests on",
```

Comment:
:thought_balloon: I think after this PR is merged we can adjust the default, see https://github.com/ipfs/someguy/issues/47